a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -363,3 +363,70 @@ def revert(self, vthunder, *args, **kwargs):
         except Exception as e:
             LOG.exception("Failed to revert partition create : %s", str(e))
             raise
+
+
+class CreateVLAN(task.Task):
```

Comment:
We shouldn't call this "CreateVLAN" that tells the operator that we are creating a neutron network of vlan type. Instead we can call it "TagEthernetInterface" as was done in the `feature/trunking` branch.

Reply:
Done.